services/validation/README.md

```diff
@@ -61,17 +175,75 @@ ConvertLib (/home/user/dir/truffle-example/contracts/ConvertLib.sol):
   Compiled with Solidity 0.5.16
   https://solc-bin.ethereum.org/wasm/soljson-v0.5.16+commit.9c3226ce.js
   https://solc-bin.ethereum.org/linux-amd64/solc-linux-amd64-v0.5.16+commit.9c3226ce
+  sourcify-validation --prepare-json /home/user/truffle-example/contracts/ConvertLib.sol:ConvertLib test/files/truffle-example.zip | solc --standard-json
 
 MetaCoin (/home/user/dir/truffle-example/contracts/MetaCoin.sol):
   Success!
   Compiled with Solidity 0.5.16
   https://solc-bin.ethereum.org/wasm/soljson-v0.5.16+commit.9c3226ce.js
   https://solc-bin.ethereum.org/linux-amd64/solc-linux-amd64-v0.5.16+commit.9c3226ce
+  sourcify-validation --prepare-json /home/user/truffle-example/contracts/MetaCoin.sol:MetaCoin test/files/truffle-example.zip | solc --standard-json`
 
 Migrations (/home/user/dir/truffle-example/contracts/Migrations.sol):
   Success!
   Compiled with Solidity 0.5.16
   https://solc-bin.ethereum.org/wasm/soljson-v0.5.16+commit.9c3226ce.js
   https://solc-bin.ethereum.org/linux-amd64/solc-linux-amd64-v0.5.16+commit.9c3226ce
+  sourcify-validation --prepare-json /home/user/truffle-example/contracts/Migrations.sol:Migrations test/files/truffle-example.zip | solc --standard-json
+
+```
+
+## Usage as a module
+* `npm install sourcify-validation`
+#### Process paths
+```typescript
+import { ValidationService } from 'sourcify-validation';
+
+const validationService = new ValidationService();
+const paths = ["path/to/file1", "path/to/file2"];
+const checkedContracts = validationService.checkPaths(paths);
```

Comment:
Can you add a comment that this does the main work?

Reply:
Sure.